deployment/keystone/changeset/append_node_capbilities.go

```diff
@@ -8,19 +8,14 @@ import (
 	"github.com/smartcontractkit/chainlink/deployment/keystone/changeset/internal"
 )
 
-var _ deployment.ChangeSet = AppendNodeCapabilities
+var _ deployment.ChangeSet[AppendNodeCapabilitiesRequest] = AppendNodeCapabilities
```

Comment:
We can use continue to use pointers in the cases that already were:
```suggestion
var _ deployment.ChangeSet[*AppendNodeCapabilitiesRequest] = AppendNodeCapabilities
```

Reply:
Yeah this is good, and should probably make it so I don't need to update chainlink-deployments. Thanks!